packages/core/src/modules/slack/use-cases/update-birthdates-to-db.ts

```diff
@@ -0,0 +1,41 @@
+import { db } from '@/infrastructure/database';
+import { slack } from '@/modules/slack/instances';
+import { RateLimiter } from '@/shared/utils/rate-limiter';
+
+const getBirthdatesRateLimiter = new RateLimiter(
+  'slack:connections:get_birthdates',
+  {
+    rateLimit: 20,
+    rateLimitWindow: 60,
+  }
+);
+
+export async function updateBirthdatesfromSlack() {
+  const members = await db
+    .selectFrom('students')
+    .select(['slackId'])
+    .where('slackId', 'is not', null)
+    .execute();
+
+  for (const member of members) {
+    await updateBirthdatebySlackId(member.slackId!);
+  }
+}
+
+async function updateBirthdatebySlackId(slackId: string) {
+  await getBirthdatesRateLimiter.process();
```

Comment:
From my understanding this line should be added here since this is right before each API call